gruntfile.js

```diff
@@ -123,6 +123,7 @@ module.exports = function(grunt) {
       }
     },
     clean: {
+      'dist': ['public/dist'],
```

Comment:
Added a clean step so that running "npm start" cleans and rebuilds the dist folder. Will fix the issue of making changes and not having them show up sometimes.